### PR TITLE
Feature 97 display metrics

### LINKED ISSUE #97: Android display metrics
As a user I would like to get the android display metrics from the device.

### PR DESCRIPTION
**Pull request by MrWalrus3451**
Resolves #97

### CODE REVIEW

File: app/src/main/java/fi/metatavu/muisti/exhibitionui/views/MuistiActivity.kt [hunk 1]
```diff
@@ -1,28 +1,46 @@
...
 
-    private var settingsClickCOunter = 0
+    private var settingsClickCounter = 0
+    private var logoutWarningToast: Toast? = null
     private val clickCounterHandler = Handler()
     protected var currentPageView: PageView? = null
     val transitionElements: MutableList<View> = mutableListOf()
+    var countDownTimer: CountDownTimer? = null
+
+
```
Comment (jarinykanen) [inline app/src/main/java/fi/metatavu/muisti/exhibitionui/views/MuistiActivity.kt:37]:
```suggestion
```

File: app/src/main/java/fi/metatavu/muisti/exhibitionui/views/MuistiActivity.kt [hunk 2]
```diff
@@ -117,13 +147,62 @@ abstract class MuistiActivity : AppCompatActivity() {
...
         val intent = Intent(this, MainActivity::class.java)
         VisitorSessionContainer.setVisitorSession(null)
         this.startActivity(intent)
+        finish()
+    }
+
+    /**
+     * Displays logout warning with a running timer
+     */
+    fun logoutWarning(timeUntilLogout: Long) {
```
Comment (jarinykanen) [inline app/src/main/java/fi/metatavu/muisti/exhibitionui/views/MuistiActivity.kt:184]: Add docs

File: app/src/main/java/fi/metatavu/muisti/exhibitionui/views/MuistiActivity.kt [hunk 3]
```diff
@@ -117,13 +147,62 @@ abstract class MuistiActivity : AppCompatActivity() {
...
+            override fun onFinish() {
+                logoutWarningToast?.cancel()
+            }
+        }.start()
+    }
+
+    /**
+     * Cancels current logout warning
+     */
+    fun cancelLogoutWarning(){
```
Comment (jarinykanen) [inline app/src/main/java/fi/metatavu/muisti/exhibitionui/views/MuistiActivity.kt:203]:
```suggestion
    fun cancelLogoutWarning() {
```

**Review by jarinykanen — CHANGES_REQUESTED** [review]
Check comments

File: app/src/main/java/fi/metatavu/muisti/exhibitionui/views/MuistiActivity.kt (hunk 2 above)
Comment (MrWalrus3451) [inline app/src/main/java/fi/metatavu/muisti/exhibitionui/views/MuistiActivity.kt:183]:
```suggestion
    /**
     * Displays logout warning with a running timer
     * @param timeUntilLogout time left until logout
     */
    fun logoutWarning(timeUntilLogout: Long) {
```

**Pull request merged**